cmd/porter/main.go

```diff
@@ -133,11 +133,17 @@ func ShouldShowUngroupedCommand(cmd *cobra.Command) bool {
 }
 
 func addBundlePullFlags(f *pflag.FlagSet, opts *porter.BundlePullOptions) {
+	addDeprecatedTagFlag(f, opts)
 	addReferenceFlag(f, opts)
 	addInsecureRegistryFlag(f, opts)
 	addForcePullFlag(f, opts)
 }
 
+func addDeprecatedTagFlag(f *pflag.FlagSet, opts *porter.BundlePullOptions) {
+	f.StringVar(&opts.Tag, "tag", "", "")
+	f.MarkDeprecated("tag", "use --reference to declare a full bundle reference")
```

Comment:
Will this print a warning so that they know it's been deprecated and that they should stop using it? If not, let's an the warning to where you do the reference = tag assignment.

Reply:
It will.  This is what it currently looks like:
```
 $ p install --tag localhost:5000/cert-manager:v0.1.0 -c mk -p cert-manager
Flag --tag has been deprecated, use --reference to declare a full bundle reference
installing cert-manager...
...
```

Do we want stronger verbiage (e.g. `WARNING ...`)?

Reply:
Nope! I just hadn't used the cobra deprecation flag and wasn't sure how it worked.   👍